src/software/firmware/srcs/pressure_controller.cpp

```diff
@@ -346,11 +352,16 @@ void PressureController::safeguardPressionCrete(uint16_t p_centiSec) {
 
 void PressureController::safeguardPressionPlateau(uint16_t p_centiSec) {
     if (m_subPhase == CycleSubPhases::HOLD_INSPIRATION) {
-        if (m_pressure >= m_maxPlateauPressureCommand) {
-            // TODO vérifier avec médical si on doit délester la pression
-            // TODO alarme franchissement plateau haut
+        if (m_pressure < ALARM_THRESHOLD_PLATEAU_UNDER_2_CMH2O) {
+            m_alarmController.detectedAlarm(24u, m_cycleNb);
         } else {
-            plateau();
+            m_alarmController.notDetectedAlarm(24u);
```

Comment:
I suggest to use the p_code instead of 24u, same for 17u

Reply:
I use `#define` for each alarm code